WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEpilogueFragment.java

```diff
@@ -64,12 +64,14 @@ protected boolean listenForLogin() {
     }
 
     @Override
-    protected @LayoutRes int getContentLayout() {
+    protected @LayoutRes
```

Comment:
The line break seems unnecessary, I wonder if we could make the checkstyle keep it on a single line. Feel free to ignore this comment, it just caught my eye, so I wanted to mention it.

Reply:
Yep, looks like a glitch in the AS config, I'll fix this double check the config.